### PR TITLE
Ch/feature/conditional rendering

### PR DESCRIPTION
**Pull request by conconartist**
#### Please review considerations below:

- [x] Follows Turing Style Guidelines
- [x] Changes generate no new warnings or errors(linter etc)
- [x] Checked code for and corrected any spelling errors
- [ ] README has been updated(if applicable)

@sertmer @letakeane (for code review)

#### What's this PR do? 

Added conditional rendering to show details for a single movie with dummy data to test for dynamic code.  Adds functionality to the "Back to Lobby" button so that when a poster is clicked, the movie details are shown and the button takes the user back to the original page view of movies when clicked.

#### Where should the reviewer start? 

New code is updated in MovieInfo.js and App.js

#### How should this be manually tested? 

Go to the source directory for this project and run the server by entering `run npm start` in your terminal.  On the browser, click on the first poster "The Devil All the Time" and the user should see only details for that movie.  Click on the button "Back to Lobby" at the top of the page under the header to be taken back to the full display of movies.

#### Screenshots (if appropriate)

Screen Shot 2021-02-05 at 6 10 29 PM (2)

### CODE REVIEW

**Comment by camaragon** [comment]
We pair programmed this together, so I was there during the development. One thing I noticed is we'll want to change the `handleChange()` method name to something more like `handleClick()` or `returnHome()` since it sends the user to main poster view on click of a button. There is currently a lack of SCSS styling as we plan to divide that work up over the weekend.

File: src/component/App.js [hunk 1]
```diff
@@ -3,21 +3,33 @@ import React, { Component } from 'react';
...
 import Movies from './Movies';
+import MovieInfo from './MovieInfo';
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: movieData.movies
+      movies: movieData.movies,
+      selectedMovie: ""
```
Comment (sertmer) [inline src/component/App.js:13]: Is the selectedMovie going to be a string data type? Looking down, it seems like it eventually is an object. Any reasoning for having it start as an empty string?

File: src/component/MovieInfo.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+              <img className='selected-movie-poster' src={selectedMovie.poster_path}></img>
+              <p className='selected-movie-overview'>{selectedMovie.overview}</p>
+              <p className='selected-movie-rating'>{selectedMovie.average_rating}</p>
+              <p className='selected-movie-date'>{selectedMovie.release_date}</p>
+              <p className='selected-movie-revenue'>{selectedMovie.revenue}</p>
+              <p className='selected-movie-runtime'>{selectedMovie.runtime}</p>
+              <p className='selected-movie-tagline'>{selectedMovie.tagline}</p>
+              <p className='selected-movie-genres'>{genreNames}</p>
+            </div>  
+        </section>
```
Comment (sertmer) [inline src/component/MovieInfo.js:22]: lots of selectedMovie in here - maybe we could destructure one more time to save us having to type it over and over again?

File: src/movieData.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 const movieData = {
     "movies": [
+      {id: 1, title: "Fake Movie Title", poster_path: "https://image.tmdb.org/t/p/original//7G2VvG1lU8q758uOqU6z2Ds0qpA.jpg", backdrop_path: "https://image.tmdb.org/t/p/original//oazPqs1z78LcIOFslbKtJLGlueo.jpg", release_date: "2019-12-04", overview: "Some overview that is full of buzzwords to attempt to entice you to watch this movie! Explosions! Drama! True love! Robots! A cute dog!", average_rating: 6, genres: [{id: 18, name:"Drama"}], budget:63000000, revenue:100853753, runtime:139, tagline: "It's a movie!" },
```
Comment (sertmer) [inline src/movieData.js:3]: I might drop all of these properties onto their own lines for readability here.

**Review by sertmer — COMMENTED** [review]
Looking good! nice mix of functional and class-based componets!

**Comment by sertmer** [comment]
No need to make any changes or respond - just some good stuff to think about going forward!

**Pull request merged**